website/src/blogs/openebs-0-5-enables-storage-policies-for-kubernetes-persistent-volumes.md

```diff
@@ -0,0 +1,74 @@
+---
+title: OpenEBS 0.5 enables Storage Policies for Kubernetes Persistent Volumes
+author: Kiran Mova
+author_info: Contributor and Maintainer OpenEBS projects. Chief Architect MayaData. Kiran leads overall architecture & is responsible for architecting, solution design & customer adoption of OpenEBS.
+date: 30-11-2017
+tags: Container, Open Source, Storage Containers, Kubernetes, Updates
+excerpt: Personally, it is very exciting and enriching to see the growth of the OpenEBS project — from its capabilities, contributors and community perspective!
+---
+
+
+Personally, it is very exciting and enriching to see the growth of the OpenEBS project — from its capabilities, contributors and community perspective!
+
+I believe the metric to measure the success of OpenSource Projects is determined by the number of users and the conversion ratio of users turning into contributors. The past couple of months (thanks to campaigns like Hacktoberfest and the ongoing OpenEBS Hackfest), we have seen a steep rise in the number of contributors and contributions to the OpenEBS project.
+![OpenEBS contributors](https://cdn-images-1.medium.com/max/800/1*BMOr9ULh_7KnM6k8aUj9hw.png)
```

Comment:
can we add blank line before and after all the images,

Reply:
Done.